src/core/egg-herbie.rkt

```diff
@@ -584,8 +587,8 @@
   (λ () (make-hash))
   (λ () (make-hash))
   (λ (cache)
-    (define ffi-rules/rule (hash-values cache))
-    (for ([ffi-rules (in-list ffi-rules/rule)])
+    (for ([(_ entry) (in-hash cache)])
+      (define ffi-rules (map cdr entry))
       (for-each free-ffi-rule ffi-rules))))
```

Comment:
Do we also need to clear the hash table?

Reply:
I don't think so that procedure is just clear the FFI stuff. The Racket GC will take care of the rest.